docs/mkdocs/docs/api/macros/nlohmann_define_type_intrusive.md

```diff
@@ -40,10 +40,12 @@ See examples below for the concrete generated code.
 
 !!! info "Prerequisites"
 
-    1. The type `type` must be default constructible. See [How can I use `get()` for non-default constructible/non-copyable types?](../../features/arbitrary_types.md#how-can-i-use-get-for-non-default-constructiblenon-copyable-types)
+    1. The type `type` must be default constructible. See [How can I use `get()` for non-default constructible/non-copyable types?][GetNonDefNonCopy]
```

Comment:
Why this indirection?

Reply:
This and all other indirections are part of "Fix some style check issues: missing admonition titles, line length, missing sections, ...", specifically line length.

Reply:
I think we should not try to reduce the line length at all costs...

Reply:
I don't think its *that* bad. :man_shrugging:

Want me to revert?

Reply:
No, keep it as is.

I'm looking forward to the day where the style of links in the documentation is the most pressing topic ;-)